metricbeat/mb/mb.go

```diff
@@ -111,6 +118,7 @@ type ModuleConfig struct {
 	MetricSets []string                `config:"metricsets" validate:"required"`
 	Enabled    bool                    `config:"enabled"`
 	Filters    processors.PluginConfig `config:"filters"`
+	Raw        bool                    `config:"raw"`
```

Comment:
It would be nice to also add a godoc comment explaining the meaning of this field.